### ISSUE TITLE
Display of negative tuning number on SPSA test

### ISSUE THREAD

**Issue by BM123499** [glinscott/fishtest#884]
it seems the Y max value can't be below 0. It's not good to see it's changing. I suppose "bigger" negative number may be harder to visualize how it's changing.

Test: https://tests.stockfishchess.org/tests/view/602fa6cc7f517a561bc49dab
![image](https://user-images.githubusercontent.com/48841672/108520847-e01d7b00-72a9-11eb-835f-f06e044fbf56.png)

**Comment by xoto10** [comment]
@ppigazzini 
Something to try for this problem, the following edit (line 19) to https://github.com/glinscott/fishtest/blob/master/server/fishtest/static/js/spsa.js might help. If not I don't think it will do any harm anyway, so seems worth trying.

```
    'height': 500,
+   'vAxis': {
+     'viewWindowMode': 'pretty'
+   },
    'legend': {
```

**Comment by xoto10** [comment]
It seems this is obviously a bug in the graphing sw, but if we can find a workaround that would be nice.

I notice that if displaying 2 negative parameters the y-axis adjusts so the lines use the full height of the graph - not sure this helps us at all, but if you have another negative parameter of roughly the same magnitude that can be displayed this may help.

**Comment by ppigazzini** [comment]
Fixed by #1319
